### PR TITLE
[FEATURE] adds new manifest param

### PR DESCRIPTION
**Pull request by rgomeztao**
[ADF-1248](https://oat-sa.atlassian.net/browse/ADF-1248)

Changelog:
* Adds a new param to manifest

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -235,4 +238,7 @@
     'containerServiceProviders' => [
         CopierServiceProvider::class,
     ],
+    'includedRoles' => [
+        TaoItemsRoles::ITEM_AUTHOR => [TaoItemsRoles::ITEM_CONTENT_CREATOR]
```
Comment (gabrielfs7) [inline manifest.php:242]: Lets also include the other roles related to ItemAuthor here (if any). We can check the mapping provided by business or ask them.

**Review by gabrielfs7 — APPROVED** [review]
Good job! 

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: manifest.php (hunk 1 above)
Comment (rgomeztao) [reply manifest.php]: new roles from editing items on https://www.taotesting.com/user-guide/managing-tao/management/managing-tao-management-atomic-roles/ have been added, thanks!

**Review by shpran — APPROVED** [review]

**Comment by rgomeztao** [comment]
this param is no longer needed as the scope of the solution has shifted to the workflow only. Closing. Thanks!

**Pull request closed without merging**